content/sensu-core/2.0/guides/install-check-executables-with-assets.md

```diff
@@ -79,9 +79,76 @@ $ sensuctl event list
   sensu-agent    check_website      CheckHttpResponseTime OK: 345      0       false    2018-04-06 20:38:34 +0000 UTC
 {{< /highlight >}}
 
+## What are handler and mutator assets?
+
+Handler and mutator assets are much like check assets. They are resources that handlers
+and mutators can specify as dependencies, and since those are executed server side, it
+is the backend that ensures all of the required assets are available during runtime.
+
+## How to create a handler that depends on an asset 
+
+### Adding an asset to a handler on creation
+
+[Create an asset][2] (ex.`sensu-influxdb-handler`), and create a handler (ex. `influx-db`).
+
+{{< highlight shell >}}
+$ sensuctl handler create influx-db \
+  --command "sensu-influxdb-handler --addr 'http://123.4.5.6:8086' --username 'foo' --password 'bar' --db-name 'myDB'" \
+  --runtime-assets sensu-influxdb-handler
+{{< /highlight >}}
+
+### Validating the handler asset
+
+Once the handler is setup, it must be attached to a check to handle the event output (ex. `collect-metrics.sh`).
```

Comment:
Is `collect-metrics.sh` the name of the check here? Would `collect-metrics` be better?

Reply:
Yes, that's right.. and yes that would be better!